.changeset/hot-tigers-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sveltejs/adapter-cloudflare-workers': patch
+'@sveltejs/adapter-cloudflare': patch
+---
+
+fix: remove `x-robots-tag: noindex` header for `_app/*` files
```

Comment:
what is the issue this is fixing, is it safe to remove?

Reply:
This was added a long time ago but it wasn’t clear why. I did some googling on it and I don’t think it does anything since search engines mostly index pages only I think? If this is useful, we should be adding it to all the adapters anyway

Reply:
can we move this to a separate PR then and find out what it was used for? Initial hunch is that crawlers hit these files and incurred cost, so removing it could have negative impact on users that still expect this to work.

Reply:
Good idea, thanks. I've reverted it in this PR